driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java

```diff
@@ -166,6 +165,26 @@ public GraphComputer compute() throws IllegalArgumentException {
             vertexIds);
     }
 
+    /**
+     * Convert a label to orientdb class name
+     */
+    public String labelToClassName(String label, String prefix) {
+    	if (configuration.getBoolean(CONFIG_LABEL_AS_CLASSNAME, false)) {
+        	return label;
+    	}
+		return label.equals(prefix) ? prefix : prefix + "_" + label;
```

Comment:
sure, please go ahead. I'm using intellij for this, but don't have hard feelings re formatting java code